routes/arrivals.js

```diff
@@ -147,8 +147,284 @@ Works like \`/stops/{id}/departures\`, except that it uses [\`hafasClient.arriva
 						content: {
 							'application/json': {
 								schema: {
-									type: 'array',
-									items: {type: 'object'}, // todo
+									'type': 'array',
+									'items': {
+										'type': 'object',
+										'properties': {
+											'tripId': {
+												'type': 'string'
+											},
+											'stop': {
```

Comment:
Let's move re-used sub schemas (e.g. a stop) into a variable each, and then use them here.